controller/trackerquery_blackbox_test.go

```diff
@@ -294,14 +297,46 @@ func (rest *TestTrackerQueryREST) TestCreateTrackerQueryID() {
 		assert.Equal(t, trackerquery.Data.ID, result.Data.ID)
 	})
 	rest.T().Run("invalid - fail", func(t *testing.T) {
-		tqpayload := newCreateTrackerQueryPayload(fxt.Spaces[0].ID, fxt.Trackers[0].ID)
+		tqpayload := newCreateTrackerQueryPayload(fxt.Spaces[0].ID, fxt.Trackers[0].ID, fxt.WorkItemTypes[0].ID)
 		invalidID := uuid.Nil
 		tqpayload.Data.ID = &invalidID
 		test.CreateTrackerqueryBadRequest(t, svc.Context, svc, trackerQueryCtrl, &tqpayload)
 	})
 }
 
-func newCreateTrackerQueryPayload(spaceID uuid.UUID, trackerID uuid.UUID) app.CreateTrackerqueryPayload {
+func (rest *TestTrackerQueryREST) TestInvalidWITinTrackerQuery() {
+	t := rest.T()
+	resource.Require(t, resource.Database)
+	rest.T().Run("nil WIT in trackerquery payload", func(t *testing.T) {
+		fxt := tf.NewTestFixture(t, rest.DB,
+			tf.SpaceTemplates(2),
```

Comment:
Here you don't need the `SpaceTemplate(2)` line.

Reply:
removed 
7eafcf3